docs/reference/ImageCms.rst

```diff
@@ -114,201 +128,193 @@ can be easily displayed in a chromaticity diagram, for example).
         One of ``ImageCms.INTENT_ABSOLUTE_COLORIMETRIC``, ``ImageCms.INTENT_PERCEPTUAL``,
         ``ImageCms.INTENT_RELATIVE_COLORIMETRIC`` and ``ImageCms.INTENT_SATURATION``.
 
-        :type: :py:class:`int`
-
     .. py:attribute:: profile_id
+        :type: bytes
 
         A sequence of 16 bytes identifying the profile (via a specially
         constructed MD5 sum), or 16 binary zeroes if the profile ID has
         not been calculated (see 7.2.18 of ICC.1:2010).
 
-        :type: :py:class:`bytes`
-
     .. py:attribute:: copyright
+        :type: Optional[str]
 
         The text copyright information for the profile (see 9.2.21 of ICC.1:2010).
 
-        :type: :py:class:`unicode` or ``None``
-
     .. py:attribute:: manufacturer
+        :type: Optional[str]
 
         The (English) display string for the device manufacturer (see
         9.2.22 of ICC.1:2010).
 
-        :type: :py:class:`unicode` or ``None``
-
     .. py:attribute:: model
+        :type: Optional[str]
 
         The (English) display string for the device model of the device
         for which this profile is created (see 9.2.23 of ICC.1:2010).
 
-        :type: :py:class:`unicode` or ``None``
-
     .. py:attribute:: profile_description
+        :type: Optional[str]
 
         The (English) display string for the profile description (see
         9.2.41 of ICC.1:2010).
 
-        :type: :py:class:`unicode` or ``None``
-
     .. py:attribute:: target
+        :type: Optional[str]
 
         The name of the registered characterization data set, or the
         measurement data for a characterization target (see 9.2.14 of
         ICC.1:2010).
 
-        :type: :py:class:`unicode` or ``None``
-
     .. py:attribute:: red_colorant
+        :type: Optional[tuple[tuple[float]]]
 
         The first column in the matrix used in matrix/TRC transforms (see 9.2.44 of ICC.1:2010).
 
-        :type: ``((X, Y, Z), (x, y, Y))`` or ``None``
+        The value is in the format ``((X, Y, Z), (x, y, Y))``, if available.
 
     .. py:attribute:: green_colorant
+        :type: Optional[tuple[tuple[float]]]
```

Comment:
`:type:` should be directly following an attribute to give a cleaner output. This automatically surrounds the value with `:py:class`. I used `Optional` from `typing` to fix the resulting reference errors where applicable.